solr/solr-ref-guide/src/replica-management.adoc

```diff
@@ -321,7 +412,42 @@ http://localhost:8983/solr/admin/collections?action=DELETEREPLICA&collection=tes
 
 Assign an arbitrary property to a particular replica and give it the value specified. If the property already exists, it will be overwritten with the new value.
 
-`/admin/collections?action=ADDREPLICAPROP&collection=collectionName&shard=shardName&replica=replicaName&property=propertyName&property.value=value`
+[.dynamic-tabs]
+--
+
+[example.tab-pane#v1addreplicaprop]
+====
+[.tab-label]*V1 API*
+
+*Input*
+
+[source,text]
+----
+http://localhost:8983/solr/admin/collections?action=ADDREPLICAPROP&collection=collectionName&shard=shardName&replica=replicaName&property=propertyName&property.value=value
```

Comment:
[0] It might be cool if the v1 and v2 examples here used the same param vals so they were truly equivalent.